python/lsst/ip/isr/isrTask.py

```diff
@@ -1607,7 +1607,9 @@ def run(self, ccdExposure, *, camera=None, bias=None, linearizer=None,
                     self.config.brighterFatterApplyGain,
                     bfGains
                 )
-            if bfResults[1] == self.config.brighterFatterMaxIter - 1:
+            bfCorrIters = bfResults[1]
+            self.metadata["LSST ISR BF CORR ITERS"] = bfCorrIters
```

Comment:
I think this should be `NITER` or maybe `N_ITER`

Reply:
Oh, I realize on my PR I call this `LSST ISR BFK` and we should make sure these are the same.  I'd rather not have an extra word here, so either `LSST ISR BFK NITER` (following DM-48783) or `LSST ISR BFCORR NITER` and I can update my PR.  (Same comment below)